testsuite/features/secondary/proxy_container_cobbler_pxeboot.feature

```diff
@@ -0,0 +1,139 @@
+# Copyright (c) 2021-2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+#
+# We also test 'Bootstrapping using the command line' in this feature with the following script:
+# https://github.com/uyuni-project/uyuni/blob/master/java/conf/cobbler/snippets/minion_script
+
+@skip_if_github_validation
+@containerized_server
+@proxy
+@private_net
+@pxeboot_minion
+@scope_cobbler
+Feature: PXE boot a terminal with Cobbler and containerized proxy
+  In order to automate client system installations with a containerized proxy
+  As the system administrator
+  I want to PXE boot one host with Cobbler
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+    And I am on the Systems overview page of this "proxy"
+
+  Scenario: Start Cobbler monitoring
+    When I start local monitoring of Cobbler
+
+  # We currently test Cobbler with SLES 15 SP4, even on Uyuni
+  Scenario: Install the TFTP boot package on the server for Cobbler tests
+    When I install package tftpboot-installation on the server
+    And I wait for "tftpboot-installation-SLE-15-SP4-x86_64" to be installed on "server"
```

Comment:
I guess this is fine for tests, but our user will need to do it differently. 
Not sure how it is currently documented, but this will not survive server image update as it is not installed in the volume.

This should be installed/extracted outside of container and then used `mgradm distro copy` command to move it to the container.

Reply:
Btw. not only image update, but mere server container restart "forgot" this installed package. That's why we don't want to allow any installation of packages in the container.

Reply:
Here is the current 5.0 documentation, it is containers-unaware:
```
.Procedure: Extracting Files from an Installation Package
. On the {productname} Server, install the package whose name starts with [package]``tftpboot-installation``.
  You can determine its exact name with the command [command]``zypper se tftpboot-installation``
. Determine where the installation files went with the command [command]``ls -d /usr/share/tftpboot-installation/*``.
  Take a note of the file path.
  You will need it when you declare the distribution to {productname}.

This procedure prepares the installation of the same operating system version as the one that powers your {productname} Server.
If you want to install a different operating system or version on the client, you need to manually get the package [package]``tftpboot-installation-*`` from the distribution it belongs to.
In the [menu]``Package Search`` input box {productname}, search the packages whose names start with [package]``tftpboot-installation``, then look at the package's details.
They show the local path below [path]``/var/spacewalk/``.
```
I will create a bug report for documentation @aaannz .

I will also try to change the test as you suggest.

Reply:
https://bugzilla.suse.com/show_bug.cgi?id=1222692 - https://bugzilla.suse.com/show_bug.cgi?id=1222692

Reply:
and new test suite code: https://github.com/uyuni-project/uyuni/pull/8466/commits/8ce1cb06255b80a509ac050793e0bfa6dea2cb2c